tools/star.wdl

```diff
@@ -124,8 +124,8 @@ task alignment {
     }
 
     output {
-        File star_log = output_prefix + "Log.final.out"
-        File star_bam = output_prefix + "Aligned.out.bam"
+        File star_log = output_prefix + ".Log.final.out"
```

Comment:
Once this is merged, I'll need to update the t-SNE workflow to supply a prefix without the dot.

Reply:
You cloud also just cherrypick the commit. Might be easier and you can do it now instead of waiting and potentially forgetting. Just my two cents.

Reply:
@adthrasher This is merged in now FYI.